python/hsfs/engine/__init__.py

```diff
@@ -39,6 +39,8 @@ def init(engine_type):
                 )
             _engine_type = "hive"
             _engine = hive.Engine()
+        elif engine_type == "hops_experiment":
+            pass
```

Comment:
how does the error look like if you try to call something that requires an engine? we could add a exception that with some info in the `get_instance()` if `_engine == "hops-experiment"`. just to avoid cryptic error messages.